app/models/user.rb

```diff
@@ -65,7 +63,12 @@ class User < ApplicationRecord
           class_name: 'InPersonEnrollment', foreign_key: :user_id, inverse_of: :user,
           dependent: :destroy
 
-  attr_accessor :asserted_attributes, :email
+  attr_accessor :asserted_attributes
+
+  # @return [String, nil]
+  def first_email
```

Comment:
I'll throw a note here that we have this method for doing something similar with phone numbers:

https://github.com/18F/identity-idp/blob/4684179b0cf5f347ba87b39ac458526d53da0f17/app/models/user.rb#L254-L256

We may be able to borrow some of the patterns from that one if we want.